views/partials/barista_shifts.ejs

```diff
@@ -0,0 +1,51 @@
+<tr data-id="<%= shift._id %>">
```

Comment:
I think this component should deliver the whole table, not just a table row.. Since the structure of your rows are tightly linked to the rest of the table (ie, if the column orders changed you'd have to change that in the headers and the rows), I think this component will be more useful if it includes the whole table. You can even include the `h2` if you want and add a `title` attribute, but that might be slightly overkill.

Reply:
Maybe I'm missing something here... but I can't think of a way to include the whole table and still make `barista_shifts.ejs`  partial table component reusable for my active shifts and past shifts. Because I'm looping through my shiftData and displaying each individual shift as a table row. If I have to put the whole table in my partial component, then I'll have to create 2 of them for active shifts and past shifts each. 

![table](https://user-images.githubusercontent.com/102274095/220242828-45b7756c-228d-4f8a-b503-a2af499c04c3.jpg)